app/templates/screwdriver.yaml

```diff
@@ -0,0 +1,6 @@
+jobs:
+  main:
+    image: node:6
+    steps:
+      - init: npm install
+      - test: npm test
```

Comment:
Yeah, I didn't want to go overboard on the templated screwdriver.yaml, since there's no guarantee the user is even using node. I've updated the real sd.yaml for this repo. I thought I had already, but apparently that was in the wrong window/repo.